resources/measure-info.json

```diff
@@ -23,6 +23,18 @@
       }
     ]
   },
+  {
+    "group_name": "2. Build Residential Schedule File",
+    "group_steps": [
+      {
+        "name": "1. Build Residential Schedule File",
+        "dependencies": "",
+        "measures": [
+          "BuildResidentialScheduleFile"
+        ]
+      }
+    ]
+  },
```

Comment:
Hmm. I'm pretty sure this file shouldn't be needed anymore. Can you try removing this file and all references to `workflow_json` in the BuildExistingModel and ApplyUpgrade (and any other?) measures?

Reply:
Are you saying it can be removed because the order of measures to be applied is basically fully dictated now?

Reply:
This workflow json was created to handle the situation where there are multiple child measures (i.e., the old BEopt OpenStudio measures) referenced in options_lookup.tsv and called by our meta-measures. We needed a way to know the order in which the measures are run. In ResStock-HPXML, we don't have that situation anymore. I think you'll find that everything works as expected without this file.

Reply:
I'm removing it on a PR directly to `restructure-v3`: https://github.com/NREL/resstock/pull/692